### PR TITLE
Updating the Info buttons

### PR DESCRIPTION
**Pull request by melyxlin**
This PR is updates the module information buttons with helpful links and a section for notes and also adds a dialog box that opens when app is rendered.

### CODE REVIEW

File: web/client/src/components/Head.jsx [hunk 1]
```diff
@@ -1,14 +1,86 @@
...
+          <Button
+            onClick={() => {
+              setOpen(false);
+            }}
+            color="primary"
+          >
+            Close
+          </Button>
+        </DialogActions>
+      </Dialog>
```
Comment (yousefa00) [inline web/client/src/components/Head.jsx:83]: This should be added to `app.js` if we want it to load initially. `Head` is for everything related to the header at the top of the page

File: web/client/src/components/info/ModuleInfo.jsx [hunk 2]
```diff
@@ -42,22 +44,45 @@ export default function ModuleInfo(props) {
         <HelpOutlineIcon />
       </IconButton>
 
-      <Dialog onClose={handleClose} open={open}>
+      <Dialog fullWidth maxWidth="sm" onClose={handleClose} open={open}>
```
Comment (LeoRuiz01) [inline web/client/src/components/info/ModuleInfo.jsx:47]: Could you change the dialog and text to appear bigger in order to accommodate to users with eyesight problems?

Comment (melyxlin) [reply web/client/src/components/info/ModuleInfo.jsx]: Done!

File: web/client/src/components/Head.jsx (hunk 1 above)
Comment (melyxlin) [reply web/client/src/components/Head.jsx]: Disabled popup when seen before.

File: web/client/src/App.js [hunk 3]
```diff
@@ -49,92 +50,125 @@ function App() {
...
+                </Grid>
+              </MaterialUI>
+            </Grid>
+          </Grid>
+        </Grid>
       </Grid>
-    </Grid>
+      {/* <MaterialUI>
+        <WelcomeBox boxOpen={true} />
+      </MaterialUI> */}
```
Comment (yousefa00) [inline web/client/src/App.js]: Don't forget to remove this if we don't need it anymore.

**Comment by yousefa00** [comment]
@googlebot I consent.

**Pull request merged**